spec/spec_helper.rb

```diff
@@ -7,6 +7,7 @@
 SimpleCov.start :rails do
   add_filter '/lib/tasks/'
   add_filter '/lib/components/'
+  add_filter '/app/policies/mission_control/'
```

Comment:
Can you toss a ticket at removing this and covering the policy in the test suite? I have set up some policy specs in my local branch and I'll be happy to work on that after shipping the UIT auth change.